packages/codegen/index.js

```diff
@@ -223,7 +223,7 @@ function createapp (argv) {
         .coerce('name', opt => {
             let name = opt.toLowerCase()
             if (name.length < 3) throw new Error('<name> is too short!')
-            if (!/^[a-z_][a-z0-9_]+$/.test(name)) throw new Error('<name> should be [a-z0-9_]+ string')
+            if (!/^[a-z_][a-z0-9-]+$/.test(name)) throw new Error('<name> should be [a-z0-9-]+ string')
```

Comment:
```suggestion
            if (!/^[a-z][a-z0-9-]+$/.test(name)) throw new Error('<name> should be [a-z0-9-]+ string')
```
![изображение](https://github.com/open-condo-software/condo/assets/52532264/8a09fd28-0741-48a2-ac2c-dc69cf9544c6)

Reply:
or is this expected behavior?

Reply:
fixed